src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -2072,6 +2072,96 @@ BOOST_AUTO_TEST_CASE(SimpleResidualLayerTest)
   delete linearB;
 }
 
+/**
+ * Simple highway module test.
+ */
+BOOST_AUTO_TEST_CASE(SimpleHighwayLayerTest)
```

Comment:
Do you think we could run a test against a mnist subset as well?

Reply:
@zoq I had made the changes. I have refactored 'FeedForwardNetworkTest' to pass model to the test and added a test with highway model. Let me know what you think.

Reply:
Great, thanks; will take a closer look at the PR in the next days.